src/plugins/borrower-profile-exp-mixin.js

```diff
@@ -32,6 +44,103 @@ export default {
 			if (version) {
 				this.isBpModalEnabled = version === 'b';
 			}
-		}
+		},
+		goToLink() {
+			this.$kvTrackEvent('borrower-profile', 'go-to-old-bp', undefined, `${this.selectedLoan?.id}`);
+			window.open(`lend/${this.selectedLoan?.id}`);
+		},
+		async loadInitialBasketItems() {
+			try {
+				const basketId = this.cookieStore.get('kvbskt');
+				if (!basketId) {
+					this.basketItems = [];
+					return;
+				}
+				const { data } = await this.apollo.query({
+					query: loanCardBasketed,
+					variables: {
+						id: 0, // dummy id since we only need basket data
+						basketId
+					},
+					fetchPolicy: 'network-only'
+				});
+				this.basketItems = data?.shop?.basket?.items?.values || [];
+			} catch (error) {
+				logFormatter(error, 'error');
+				this.basketItems = [];
+			}
+		},
+		addToBasket(lendAmount) {
```

Comment:
Not sure why `addToBasket` is triggering `PostCheckoutAchievements`, but that query locally is being called after ATB and is returning the error below. It looks like this query isn't being triggered on dev after, so not sure specifically the source of this error, but it is a difference from dev.

```[
  {
    "message": "invalid type for variable: 'loanIds'",
    "extensions": {
      "name": "loanIds",
      "code": "VALIDATION_INVALID_TYPE_VARIABLE"
    }
  }
]
```